src/main/java/ua/softserve/rv_028/issuecitymonitor/dao/UserDao.java

```diff
@@ -1,15 +1,21 @@
 package ua.softserve.rv_028.issuecitymonitor.dao;
 
-import org.springframework.data.repository.CrudRepository;
-import ua.softserve.rv_028.issuecitymonitor.entity.User;
 
 
 import java.util.List;
-import java.util.Set;
 
-public interface UserDao extends CrudRepository<User, Long>{
+import org.springframework.data.repository.CrudRepository;
+
+
+import ua.softserve.rv_028.issuecitymonitor.entity.User;
+import ua.softserve.rv_028.issuecitymonitor.entity.enums.UserRole;
+
+import java.util.List;
 
+public interface UserDao extends CrudRepository<User, Long> {
+    List<User> findAllByOrderByIdAsc();
     List<User> findAll();
-    User findUserByUsername(String username);
 
+    User findUserByUsername(String username);
+    Long countByUserRole(UserRole userRole);
```

Comment:
You could count users by admin properly, other parameters different from UserRole.ADMIN is not used